### PR TITLE
Add code for ransac

### PR DESCRIPTION
**Pull request by jasmainak**

### CODE REVIEW

File: examples/plot_ransac.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                verbose=False, detrend=0, preload=True)
+
+###############################################################################
+# First, we set up the function to compute the sensor-level thresholds.
+from autoreject import Ransac
+from autoreject.utils import interpolate_bads
+
+ransac = Ransac(ch_type='meg', verbose='tqdm')
+ransac.fit_predict(epochs)
+epochs_clean = epochs.copy()
```
Comment (agramfort) [inline examples/plot_ransac.py]: API is weird. The output of a predict is usually used. It should be a fit or just not operate in place.

File: examples/plot_ransac.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    ax.tick_params(axis='x', which='both', bottom='off', top='off')
+    ax.tick_params(axis='y', which='both', left='off', right='off')
+
+ylim = dict(grad=(-170, 200))
+evoked.pick_types(meg='grad', exclude=[])
+evoked.plot(exclude=[], axes=axes[0], ylim=ylim, show=False)
+axes[0].set_title('Before autoreject')
+evoked_clean.pick_types(meg='grad', exclude=[])
+evoked_clean.plot(exclude=[], axes=axes[1], ylim=ylim)
+axes[1].set_title('After autoreject')
```
Comment (agramfort) [inline examples/plot_ransac.py]: Ransac not autoreject

File: examples/plot_ransac.py (hunk 1 above)
Comment (jasmainak) [inline examples/plot_ransac.py]: okay thanks for taking a look :) I was thinking something is off, but I couldn't place my finger on it. Now I know. I'll fix it directly in master tomorrow.

File: examples/plot_ransac.py (hunk 2 above)
Comment (jasmainak) [inline examples/plot_ransac.py]: Fixed here: https://github.com/autoreject/autoreject/commit/e9de2efc292efa4a486bafcf0e8ea27307793f30

**Pull request merged**